eth/tracers/tracer.go

```diff
@@ -100,13 +100,13 @@ func (mw *memoryWrapper) slice(begin, end int64) []byte {
 	if end < begin || begin < 0 {
 		// TODO(karalabe): We can't js-throw from Go inside duktape inside Go. The Go
 		// runtime goes belly up https://github.com/golang/go/issues/15639.
-		log.Warn("Tracer accessed out of bound memory", "offset", begin, "end", end)
+		log.Warn("JSTracer accessed out of bound memory", "offset", begin, "end", end)
```

Comment:
```suggestion
		log.Warn("Tracer accessed out of bound memory", "offset", begin, "end", end)
```
I think all of these should be reverted, my bad for making these changes